src/Analysis/Ast/Impl/Types/ParameterInfo.cs

```diff
@@ -34,6 +35,9 @@ public ParameterInfo(PythonAst ast, Parameter p, IPythonType type, IMember defau
             DefaultValue = defaultValue;
             Type = type;
             Kind = kind ?? ParameterKind.Normal;
+#if DEBUG
+            Debug.Assert(Kind != ParameterKind.PositionalOnlyMarker);
```

Comment:
Don't have to `#if` it, AFAIK `Debug.` is noop in release.

Reply:
Yeah, I was unsure if its arguments would be evaluated, but after some searching it looks like the compiler will remove all of the code, including anything that'd normally be run to do the arguments. I just didn't want this traffic-heavy area to do this comparison all the time.